pages/advanced/passkeys/tutorials/react-native.mdx

```diff
@@ -0,0 +1,266 @@
+
+{/* <!-- vale off --> */}
+
+<Tabs items={['iOS', 'Android']}>
+  <Tabs.Tab>
+    - Login or subscribe to the Apple Developer Program and [locate your Team
+    ID](https://developer.apple.com/help/account/manage-your-team/locate-your-team-id/).
+    - Add your Team ID instead of `add_your_apple_team_id_here` in the
+    `apple-app-site-association` file in the AASA server.
+    - Add your provisioning profile to sign the app through Xcode. You can open the generated iOS project in Xcode (open the folder `/ios` in your project) and
+    under `Signing & Capabilities` choose your team. Make sure that the `Bundle Identifier` is `com.safeglobal.reactnativepasskeys`.
+    - Activate FaceId. Go to Features > Face ID > Enrolled in the iOS simulator menu.
+  </Tabs.Tab>
+  <Tabs.Tab>
+    - Add a Google account in the settings of the Android emulated phone.
+    - Get an Android keystore SHA 256 fingerprint with this command (from the root
+    of your project):
+```bash
+keytool -list -v -keystore ./android/app/debug.keystore -alias androiddebugkey -storepass android -keypass android
+```
+    - Copy the value in front of `SHA 256` at the place of
+    `add_your_sha256_cert_fingerprints_here` in the assetslinks.json file.
+  </Tabs.Tab>
+</Tabs>
+
+{/* <!-- vale on --> */}
+
+You can now start the server with:
+
+```bash
+cd aasa-server
+npm install
+npm start
+```
+
+Once you have it running, you need a public domain secured with SSL to test the passkeys (`localhost` is not valid). You can use a service like [ngrok](https://download.ngrok.com/mac-os) to create a tunnel to your local server.
+
+After the installation, create a tunnel to the localhost port you are using, which should be `3000` if you are using the provided server.
+
+```bash
+ngrok http 3000
+```
+
+The ngrok service will provide you with a public domain. You must copy this domain (without `https://`) and add it to your project files after completing the steps below:
+
+- [passkeys.ts](https://github.com/5afe/react-native-passkeys-tutorial/blob/913cc17f353062bf1c4c55ad303e8623b6035a6b/lib/passkeys.ts#L8)
+- [app.json](https://github.com/5afe/react-native-passkeys-tutorial/blob/913cc17f353062bf1c4c55ad303e8623b6035a6b/app.json#L19). It's important to have the [developer mode](https://developer.apple.com/documentation/xcode/configuring-an-associated-domain) activated (?mode=developer) in your `app.json`.
+
+Not updating this information will result in an error when trying to create a passkey.
+
+```
+Error Domain=com.apple.AuthenticationServices.AuthorizationError Code=1004 "(null)"
+```
+
+If you are using iOS, you will also need to add this URL to the app's associated domains. Open Xcode again, in the tab `Signing & Capabilities`, click the button `+ Capability`, select `Associated Domain`, and add the ngrok URL with the prefix `webcredentials:`. For example:
+
+```bash
+webcredentials:your-ngrok-domain.ngrok-free.app
```

Comment:
?mode=developer is required at the end of the domain name but this is nor needed if made properly in the steps about the app.json above